lib/matplotlib/artist.py

```diff
@@ -1490,7 +1490,12 @@ def is_alias(self, o):
         ds = inspect.getdoc(o)
         if ds is None:
             return False
-        return ds.startswith('Alias for ')
+        result = ds.startswith('Alias for ')
+        alt_result = (o._artist_property.is_alias
+                      if hasattr(o, '_artist_property')
+                      else False)
+        assert result == alt_result
+        return result
```

Comment:
This code segment is temporary. I've written it to easily verify that the way of obtaining *is_alias* via the property is equivalent to getting that information from the docstring. Note that this is already run at import time on all artists via `ArtistInspector.__init__` -> `ArtistInspector.get_aliases()` because we inspect all artists for kwdocs. So if the assert holds in the tests the equivalence is proven.

I propose the following review pattern:

After two people have seen that the tests pass and have approved the PR, I'll change this whole function to only using the new way:
   ```
   def def is_alias(self, o):
       return (o._artist_property.is_alias
               if hasattr(o, '_artist_property')
               else False)
   ```

PLEASE REVIEW, BUT DO NOT MERGE after the second approval.

Reply:
Would it make sense to add that "Alias for..." part of the doc-string based on this? So not only provide an `is_alias` boolean, but rather a string for which artist property it is an alias for?